docs/en_US/operation/usage/monitor_with_prometheus.md

```diff
@@ -80,6 +80,10 @@ After version 2.0.0, we added connection-related metrics for source/sink.
 
 The numeric types of these metrics can all be monitored using Prometheus. In the next section we will describe how to configure the Prometheus service in eKuiper.
 
+View CPU running metrics for a rule
+
+- kuiper_rule_cpuUsage: The CPU running indicator of the rule represents the CPU time used by the CPU in the past 30 seconds, in ms.
```

Comment:
Is it better to comply with the metrics naming pattern (https://prometheus.io/docs/practices/naming/): kuiper_rule_cpu_ms?

Reply:
Doc is not updated